sources/model/GGRealTimeMontior.m

```diff
@@ -792,9 +804,11 @@ - (void) socketIODidConnect:(SocketIOClient *)socketIO {
     if (self.socketIOConnectedBlock) {
         
         NSLog(@"\t\thandling connect success");
-        
         self.socketIOConnectedBlock(YES, nil);
         self.socketIOConnectedBlock = nil;
+        NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                       self.developerToken, @"developer_access_token",
```

Comment:
ok this is becoming confusing. let's talk about this comment when i get to the office